akka-cluster-sharding/build.sbt

```diff
@@ -0,0 +1,18 @@
+import akka.{ AkkaBuild, Dependencies, Formatting, MultiNode, Unidoc, OSGi }
+import com.typesafe.tools.mima.plugin.MimaKeys
+
+AkkaBuild.defaultSettings
+
+Formatting.formatSettings
+
+Unidoc.scaladocSettingsNoVerificationOfDiagrams
+
+Unidoc.javadocSettings
+
+OSGi.clusterSharding
```

Comment:
sounds good to me, same thing with dependencies

please create a separate issue